### PR TITLE
[ZT] custom DLP profiles

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 1]
```diff
@@ -7,25 +7,45 @@ layout: single
 
 # Data Loss Prevention
 
+{{<Aside type="note">}}
+Data Loss Prevention is only available on Enterprise plans.
```
Comment (cloudflare-noelle) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]: Data Loss Prevention is only available as an add-on to Zero Trust Enterprise plans

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 2]
```diff
@@ -7,25 +7,45 @@ layout: single
...
-{{</Aside>}}
 4. Select **Save profile**.
 
+### Build a custom profile
+
+1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), go to **Gateway** > **DLP Profiles**.
+2. Select **Create Profile**.
+3. Enter a name and optional description for the profile.
+4. Select **Add detection entry** and give it a name.
+5. In **Value**, enter a regular expression that defines the text pattern you want to detect.
```
Comment (cloudflare-noelle) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]: In **Value**, enter a regular expression that defines the text pattern you want to detect. Regexes are written in Rust, and we recommend validating your regex with [Rustexp](https://rustexp.lpil.uk/).

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 3]
```diff
@@ -7,25 +7,45 @@ layout: single
...
+
+    For example, `test\d\d` will detect the word `test` followed by 2 digits. We recommend validating your regex with [Rustexp](https://rustexp.lpil.uk/).
```
Comment (cloudflare-noelle) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]: Then can remove the "We recommend..."

**Review by cloudflare-noelle — CHANGES_REQUESTED** [review]
Looks great, just a couple small suggestions based on some previous questions you had asked, then good to go!

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]:
```suggestion
Data Loss Prevention is only available as an add-on to Zero Trust Enterprise plans.
```

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 4]
```diff
@@ -7,25 +7,45 @@ layout: single
...
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]:
```suggestion
    For example, `test\d\d` will detect the word `test` followed by 2 digits.
```

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 5]
```diff
@@ -7,25 +7,45 @@ layout: single
...
-{{</Aside>}}
 4. Select **Save profile**.
 
+### Build a custom profile
+
+1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), go to **Gateway** > **DLP Profiles**.
+2. Select **Create Profile**.
+3. Enter a name and optional description for the profile.
+4. Select **Add detection entry** and give it a name.
+5. In **Value**, enter a regular expression that defines the text pattern you want to detect.
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]:
```suggestion
5. In **Value**, enter a regular expression that defines the text pattern you want to detect. Regexes are written in Rust, and we recommend validating your regex with [Rustexp](https://rustexp.lpil.uk/).
```

**Review by cloudflare-noelle — APPROVED** [review]

**Review by kodster28 — APPROVED** [review]

**Pull request merged**